project-base/docker/php-fpm/Dockerfile

```diff
@@ -115,3 +115,15 @@ RUN composer install --optimize-autoloader --no-interaction --no-progress --dev
 RUN php phing composer-dev npm dirs-create test-dirs-create assets standards tests-static tests-acceptance-build
 
 RUN ./bin/console shopsys:environment:change prod
+
+########################################################################################################################
+
+FROM base as google_cloud
```

Comment:
It seems to me that ideally the Docker image should be the same, only its configuration (via ENVs or configmaps) should change.

Alternatively, the GCS adapter could require a change in the codebase, similar to installing a plugin. This way we could have Shopsys Framework independent of `superbalist/flysystem-google-storage` and `google/cloud-storage` (which might be important when we support more cloud providers, now it's not necessary imo).

Reply:
seems like dirs-create is not needed also for production so we can use production target with different `RUN ./bin/console shopsys:environment:change `